packages/components/src/menu-item/style.scss

```diff
@@ -33,14 +33,12 @@
 	// Colorize plugin icons to ensure contrast and cohesion, but allow plugin developers to override.
 	svg,
 	svg * {
-		stroke: $dark-gray-500;
 		fill: $dark-gray-500;
 	}
 
 	&:hover svg,
 	&:hover svg * {
-		stroke: $dark-gray-900 !important;
-		fill: $dark-gray-900 !important;
+		fill: $dark-gray-900;
```

Comment:
❤️

Reply:
These !importants were actually intentional. 

Try installing https://wordpress.org/plugins/dropit/, you'll note that it adds plugin icons up there that are colored. The only way we can reliably recolorize those on mouse hover and in particular on active, is by using `!important`. This is unfortunate, but it is also exactly the kind of edgecase that I believe the modifier was designed for.

Perhaps we should add a comment?

Reply:
This is how it should look:

![how it should work](https://user-images.githubusercontent.com/1204802/43756632-cd47414c-9a15-11e8-8380-823c74a33adf.gif)

This is how it will look if we remove the !important's:

![if importants are removed](https://user-images.githubusercontent.com/1204802/43756636-d2fa9be8-9a15-11e8-9d1d-797b7e8b52db.gif)

Note that the toggled state still works in the Drop It plugin even if the !important's are removed, but this is because Drop it is built by our own Riad so he was careful to make that work. In other words, he added his own important styles in his plugin. But we can't rely on plugin developers to do this, and given SVGs devs will use for plugin icons commonly rely on inline styles, I see no other way.

Reply:
> Perhaps we should add a comment?
 
👍

Reply:
I've added the !important back in for the fill style and added a comment.

I can't add the stroke style, as it makes our dashicons look fuzzy:
![icons-fuzzy](https://user-images.githubusercontent.com/677833/43760855-ea30a6b4-9a55-11e8-840e-4924bfa4cfb7.gif)

Reply:
That's a great catch. This needs to be scoped better. The only reason those rules exist for the dropdown menu is so as to catch these plugin-added icons:

![screen shot 2018-08-07 at 09 26 56](https://user-images.githubusercontent.com/1204802/43761031-1d7c860a-9a24-11e8-8375-ae015e62e152.png)

There doesn't appear to be any specific rule we can target, alas:

![screen shot 2018-08-07 at 09 27 56](https://user-images.githubusercontent.com/1204802/43761070-36294a76-9a24-11e8-810f-db7a559de7ca.png)

Should we make one?

Alternately, perhaps it's okay to remove the !important rules after all, since it's less important that the hover and active styles in the _more menu_ are colorized than it is in the toolbar.

Reply:
Ah - thanks, I didn't spot them there in menu-item form as well.

I can create a separate issue - it does look like a specific way to target the plugins would be the best trade-off.